filebeat/module/mongodb/_meta/docs.asciidoc

```diff
@@ -13,7 +13,7 @@ include::../include/gs-link.asciidoc[]
 [float]
 === Compatibility
 
-The +{modulename}+ module was tested with logs from versions v3.2.11 on Debian.
+The +{modulename}+ module was tested with plaintext logs from version v3.2.11 on Debian and json logs from version v4.4.4 on Ubuntu.
```

Comment:
Thanks for updating the docs! You will need to run `mage update` from the `filebeat` directory and commit the changes after these changes.

Reply:
done https://github.com/elastic/beats/pull/24774/commits/9823ac8a8ce4976a3cb6ff2b606dff62de67108b